src/autopas/options/IteratorBehavior.h

```diff
@@ -13,32 +13,55 @@
 namespace autopas {
 inline namespace options {
 /**
- * Class representing the acquisition function choices for the Bayesian search.
+ * Class representing the choices for behaviors of iterators.
+ * Choices are a bit vector and can thus be combined via a logical OR.
  */
 class IteratorBehavior : public Option<IteratorBehavior> {
  public:
   /**
-   * Different possibilities for iterator behaviors
+   * Type used for the internal enum.
    */
-  enum Value {
+  using Value_t = unsigned int;
+
+  /**
+   * Different possibilities for iterator behaviors.
+   */
+  enum Value : Value_t {
+    /**
+     * Iterate only over owned particles.
+     */
+    owned = 0b0001,
     /**
      * Iterate only over halo particles.
      */
-    haloOnly,
+    halo = 0b0010,
     /**
-     * Iterate only over owned particles.
+     * Iterate over both halo and owned particles. Defined fore ease of access.
      */
-    ownedOnly,
+    ownedOrHalo = 0b0011,
     /**
-     * Iterate over both halo and owned particles.
+     * Iterate only over dummy particles.
      */
-    haloAndOwned,
+    dummy = 0b0100,
     /**
-     * Iterate over both halo and owned particles and also dummy particles.
+     * Iterate over both halo and owned particles and also dummy particles. Defined fore ease of access.
      */
-    haloOwnedAndDummy,
+    ownedOrHaloOrDummy = 0b0111,
```

Comment:
Could theoretically be:
```suggestion
    ownedOrHaloOrDummy = owned | halo | dummy,
```
No idea which is less error-prone...